README.md

```diff
@@ -26,3 +26,54 @@
     pip install -r dev-requirements.txt
     ```
 
+## Howto
+Here is instruction how to create you own DVC project by yourself.
+
+0. Make sure you destroy our DVC workspace in repository:
+    ```bash
+    dvc destroy
```

Comment:
Can we change it to `dvc remote remove & dvc remote add` to show how to change the remote?

Reply:
No

Reply:
No, because it's instruction how to setup the DVC project from scratch. Instructions how to create users own remote storage to contribute to the project ar described in *how to contribute* section